components/clarinet-deployments/src/lib.rs

```diff
@@ -387,8 +415,20 @@ pub async fn generate_default_deployment(
 
     let mut repl_settings = manifest.repl_settings.clone();
     repl_settings.remote_data.enabled = false;
+
+    // Custom boot contracts are only supported on simnet
```

Comment:
Remove, I think the comment only describes the `if` below without giving additional information